lms/services/email_preferences.py

```diff
@@ -80,16 +78,8 @@ def _url(self, route, h_userid, tag):
             _query={"token": self._encode_token(TokenPayload(h_userid, tag))},
         )
 
-    def unsubscribe(self, h_userid, tag):
+    def unsubscribe(self, h_userid):
```

Comment:
The `tag` argument is no longer needed: tags no longer go in the DB in any way. The unsubscribes in the DB are still scoped to the instructor email digests though: they update user preferences called `"instructor_email_digests.days.*"`